core/src/test/java/io/questdb/test/cairo/mv/CreateMatViewTest.java

```diff
@@ -149,6 +149,58 @@ public void testCreateMatViewBaseTableDoesNotExist() throws Exception {
         assertNull(getMatViewDefinition("testView"));
     }
 
+    @Test
```

Comment:
I'd add some tests with joins and a test involving multiple symbol columns with different capacities. WDYT?